eng/Directory.Build.Common.props

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <IsMgmtLibrary Condition="'$(IsMgmtLibrary)' == '' and $(MSBuildProjectName.StartsWith('Azure.ResourceManager'))">true</IsMgmtLibrary>
     <IsMgmtSubLibrary Condition="'$(IsMgmtSubLibrary)' == '' and $(MSBuildProjectName.StartsWith('Azure.ResourceManager.'))">true</IsMgmtSubLibrary>
+    <IsMgmtCoreLibrary Condition="'$(IsMgmtCoreLibrary)' == '' and '$(MSBuildProjectName)' == 'Azure.ResourceManager'">true</IsMgmtCoreLibrary>
```

Comment:
What do you intend to use this new property for?

Reply:
I need to use batch task to generate different pieces of code with different namespaces under different folders for `Azure.ResourceManager` package (the [autorest.md](https://github.com/Azure/azure-sdk-for-net/blob/dd90d84da112ca4a2b3c53cca213b02c3af781e0/sdk/resourcemanager/Azure.ResourceManager/src/autorest.md) I'm working on). Currently, in `GenerateCode` target, the autorest command provides `--namespace` and `--output-folder` values which override the values I put in autorest.md. I'm adding a condition check that if the package is `Azure.ResourceManager`, the autorest command will not append `--namespace` and `--output-folder` in this [PR](https://github.com/Azure/autorest.csharp/pull/1709/files).

Reply:
Instead of specifying it in this way why not just add another property that is specific to this setting and then set that in Azure.ResourceManager project?

Reply:
Good advice! I have updated the PR accordingly.